pallets/ethereum-xcm/Cargo.toml

```diff
@@ -4,7 +4,7 @@ authors = [ "Parity Technologies <admin@parity.io>" ]
 description = "Xcm Transact compatibility for pallet-etherum."
 edition = "2021"
 license = "Apache-2.0"
-repository = "https://github.com/purestake/moonbeam/"
+repository = "https://github.com/moonbeam-foundation/moonbeam/"
```

Comment:
Isn't this being handled in a different PR? If we really want to handle it here, make sure we do it consistently